docs/sources/about.md

```diff
@@ -61,12 +68,10 @@ You should run Static mode when:
 
 ### Static mode Kubernetes operator
 
-The [Static mode Kubernetes operator][] is a variant of Grafana Agent first
-introduced on June 17, 2021. It is currently in beta.
+The [Static mode Kubernetes operator][] is a variant of Grafana Agent introduced on June 17, 2021. It's currently in beta.
 
-The Static mode Kubernetes operator was introduced for compatibility with
-Prometheus Operator, allowing static mode to support resources from Prometheus
-Operator, such as ServiceMonitors, PodMonitors, and Probes.
+The Static mode Kubernetes operator provides compatibility with Prometheus Operator,
+allowing static mode to support resources from Prometheus Operator, such as ServiceMonitors, PodMonitors, and Probes.
```

Comment:
Sorry for scope creep, but the "you should run Static mode Kubernetes operator" section doesn't feel right anymore, since Flow mode does the one thing listed in the bullet point. 

Should we change this to something like "It is no longer recommended to use Static mode Kubernetes operator for new deployments of Grafana Agent?"

cc @richardqlam

Reply:
I would leave the existing language and then add what @rfratto recommended below. So something like, "As of xyz date it is recommended to use Grafana agent Flow mode to support xyz use cases"

Reply:
@rfratto @richardqlam How about a note that says something like:

```
Grafana Agent version 0.37 and newer provides Prometheus Operator compatibility in Flow mode. You should run Grafana Agent Flow mode for all new Grafana Agent deployments.
```

We could add something similar to the Static section as well - the first step in nudging people to use Flow over Static or Operator.

Reply:
I like that 👍

Reply:
Added in a new commit